### ISSUE TITLE
EC bestiary: Giant Aukashungi and Xilvirek have missing attack traits

### ISSUE THREAD

**Issue by cswendrowski** [foundryvtt/pf2e#1192]
In GitLab by @Ustin on Mar 19, 2021, 03:02

Giant Aukashungi: 
* Jaws should have _reach 10 feet_
* Horn should have _deadly 2d8_ and _reach 15 feet_

Xilvirek:
* Jaws should have _chaotic, evil, magical_
* Claw should have _agile, chaotic, evil, magical, reach 10 feet_
* Tail should have _chaotic, evil, magical, reach 15 feet, versatile P_

Links: [Giant Aukashungi](https://pf2.easytool.es/index.php?id=6594&name=giant_aukashungi), [Xilvirek](https://pf2.easytool.es/index.php?id=6604&name=xilvirek)
